runtime.go

```diff
@@ -550,9 +570,33 @@ type ComponentScript struct {
 	Name string
 	// Function to render.
 	Function string
-	// Call of the function in JavaScript syntax, including parameters.
+	// Call of the function in JavaScript syntax, including parameters, and ensures parameters are HTML escaped.
 	// e.g. print({ x: 1 })
 	Call string
+	// Call of the function in JavaScript syntax, including parameters. Doesn't not HTML escape parameters; useful for directly calling.
+	// e.g. print({ x: 1 })
+	CallInline string
+}
+
+var _ Component = ComponentScript{}
+
+func (c ComponentScript) Render(ctx context.Context, w io.Writer) error {
```

Comment:
Very clever! 👍🏻 💯

Reply:
Luckily I found that this approach could be used pretty quickly from some testing :smile: